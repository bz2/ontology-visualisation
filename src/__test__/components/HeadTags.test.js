import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import HeadTags from '../../components/HeadTags';

const setup = () => {
  const props = {
    title: 'Test',
    description: 'Another test'
  };

  const component = shallow(<HeadTags {...props} />);

  return {
    component,
    props
  };
};

describe('HeadTags', () => {
  it('should match snapshot ', () => {
    const {
      component
    } = setup();

    expect(toJson(component)).toMatchSnapshot();
  });
});
