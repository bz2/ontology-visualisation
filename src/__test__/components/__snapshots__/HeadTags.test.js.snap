// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeadTags should match snapshot  1`] = `
<Head>
  <title>
    Highways England Ontology Visualisation
     - Test
  </title>
  <meta
    content="Another test"
    name="description"
  />
  <meta
    content="initial-scale=1.0, width=device-width"
    name="viewport"
  />
</Head>
`;
